<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
